app/controllers/planning_applications/assessment/consultees_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module PlanningApplications
+  module Assessment
+    class ConsulteesController < ApplicationController
+      include CommitMatchable
+
+      before_action :set_planning_application
+      before_action :set_consultation
+
+      def index
+        respond_to do |format|
+          format.html
+        end
+      end
+
+      def check
+        if @consultation.create_consultees_review
+          redirect_to planning_application_assessment_tasks_path(@planning_application)
+        else
+          flash.now(alert: "Consultees were not marked as checked for this application")
```

Comment:
Do you also need to render :index, or does it work without that?

Reply:
I *think* it works without, but will check! May add anyway for safety